src/components/InviteLearnersModal/index.jsx

```diff
@@ -97,46 +98,92 @@ class InviteLearnersModal extends React.Component {
       <>
         {submitFailed && this.renderErrorMessage()}
         <form onSubmit={e => e.preventDefault()}>
-          <p>Unassigned licenses: {availableSubscriptionCount}</p>
+          <p>
+            <FormattedMessage
+              id="admin.portal.invite.learners.modal.unassigned.licenses"
```

Comment:
nit: It would be better to use the following naming pattern for IDs
```
id="adminPortal.inviteLearnersModal.unassignedLicenses.message"
```
i.e. Use camelCase for pages/components and separate page/component name using full stop `.`

Please do this for the rest of the places as well.

Reply:
@saleem-latif I believe it would be very time-consuming to rename each ID given that there are 27 files and numerous strings. Since we decided on this convention after I completed my current work, I will apply it to the other components I start working on.